src/panels/config/entities/ha-config-entities.ts

```diff
@@ -724,6 +761,13 @@ export class HaConfigEntities extends SubscribeMixin(LitElement) {
     navigate(this, window.location.pathname, true);
   }
 
+  private _showAll() {
+    this._showDisabled = true;
+    this._showReadOnly = true;
+    this._showUnavailable = true;
+    navigate(this, window.location.pathname, true);
```

Comment:
Why is this?

Reply:
That is to react to the "Show All" button in the box that informs the user about the hidden entities. If the button is clicked we set all checkboxes in the filter menu and navigate to the same page again, where then all entities are visible.

Reply:
But why do we need to navigate to the same page again? Setting the properties should update the data?

Reply:
I had rather blindly copied that from `private _clearFilter()`, but you are right: It works fine without.